feature: add docs to composite credential (#111)
UnumID/UnumID.github.io: docs/issuance-guide.mdx

```text title="Sandbox API Key"
5YugDQgjPn2LG2xTAs/+c6Kfrf7Ie45PIGjJurP8vNk=
```

```text title="Sandbox URL"
https://core-api.sandbox-verifiedinc.com
```

```text title="Sandbox User Email"
richard@piedpiper.net
```

```text title="Sandbox User Phone"
+10123456789
```

```text title="Sandbox User Wallet Authentication OTP"
111111
```

:::

## Motivations

> As a brand that issues credentials, Verified Inc. is willing to cover your identity verification costs in order to create a truly win-win-win scenario for you, your partners, and your users. Simply put, you issue a digital ID card to any of your users (new or existing) with verified identity data, and we pay you for it. The core motivations for such an arrangement are multifaceted:

Reviewer: ```suggestion
If you issue digital ID cards, Verified Inc. may be willing to cover your identity verification costs in order to create a truly win-win-win scenario for you, your partners, and your users. Simply put, you issue a digital ID card to any of your users (new or existing) with verified identity data, and we pay you for it. The core motivations for such an arrangement are multifaceted:
```

      reimburses Hooli for the cost of the identity verification performed by
      Hooli for Richard.
    </p>
    <iframe
      src='https://marvelapp.com/prototype/1938729e/screen/90752160?emb=1&iosapp=false&frameless=false'
      width='453'
      height='864'
      allowTransparency='true'
      frameborder='0'
    ></iframe>
  </details>
- After User Sign Up
  <details>
    <summary>Demo of After User Sign Up</summary>
    <p>
      Richard has just completed Hooli account sign up which included identity
      verification via a third party service of Hooli's choosing. In the terms
      and service section Richard agrees to being issued an Hooli branded
      Verified Inc. digital ID card. Hooli issues the card with the verified
      identity data and Verified Inc. reimburses Hooli for the cost of the
      identity verification performed by Hooli for Richard.
    </p>
    <iframe
      src='https://marvelapp.com/prototype/9e98j88/screen/90752101?emb=1&iosapp=false&frameless=false'
      width='453'
      height='864'
      allowTransparency='true'
      frameborder='0'
    ></iframe>
  </details>
- Marketing Email
  <details>
    <summary>Demo of Marketing Email</summary>

>     <p>

Author: I agree, I'll open tickets for this today
